packages/dashboard-hardhat-plugin/README.md

```diff
@@ -0,0 +1,58 @@
+# @truffle/dashboard-hardhat-plugin
+
+Use [Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/) with your Hardhat projects.
+
+## What
+
+This plugin brings [Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/) to your Hardhat projects. It extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling you to sign transactions with your browser-based wallet. No more floating private keys!
```

Comment:
This functionality already exists, but without this plugin, users will only see raw requests. This plugin enables decoded results. We might want to highlight this, maybe even with like a before/after screenshot :)

Reply:
Added :) Viewable [here](https://github.com/trufflesuite/truffle/tree/feat/dashboard-hardhat-plugin/packages/dashboard-hardhat-plugin) if you want to review.